rails-commenting.rb

```diff
@@ -39,15 +41,15 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # 6)
+      # 6) Redirects the browser to that specific blog post URL. The blog_post_path is a route that returns back to the blog post page.
       redirect_to blog_post_path(@post)
     end
   end
 
-  # 7)
+  # 7) Private creates methods that are internal use only, used within the class definition. 
```

Comment:
Aye, again, I'd want to mention more about why it's private and what the use of private is.